config/default.json

```diff
@@ -55,7 +55,8 @@
     "showContactConfirmationQuestion": false
   },
   "cookies": {
-    "session": "blf-alpha-session"
+    "session": "blf-alpha-session",
+    "expiryMs": 7200000
```

Comment:
I'd move this config elsewhere, the reads like there is a cookie named `7200000`. Would be good to name it more explicitly for session expiry